analyzer/runtime/runtime.go

```diff
@@ -66,11 +71,11 @@ func NewRuntimeAnalyzer(nodeCfg config.NodeConfig, cfg *config.BlockBasedAnalyze
 		return nil, err
 	}
 
-	id, err := emerald.ID(network)
+	id, err := runtime.ID(network)
```

Comment:
I tested running multiple runtime analyzers (emerald and sapphire) in parallel using this PR. It was a slightly fake setup: I changed this line so that the "sapphire" analyzer was really pulling emerald data. There were no changes in regression tests. :tada: So it looks like we're fully ready for the sapphire impl.

I'll post the tests as top-level comments for visibility.

Reply:
cool, and I tested with the public node on sapphire mainnet